sweep/sweeper_test.go

```diff
@@ -1338,7 +1339,7 @@ func TestLockTimes(t *testing.T) {
 
 	// Check the sweeps transactions, ensuring all inputs are there, and
 	// all the locktimes are satisfied.
-	for i := 0; i < numSweeps; i++ {
+	for i := 0; i < numSweeps+1; i++ {
```

Comment:
🧐

Reply:
Why do we have numSweeps+1 loops here?

Reply:
it's a hacky way to fix things - by default the test uses `MaxInputsPerTx` of 3, and it updates the value via `ctx.sweeper.cfg.MaxInputsPerTx = 100`m which won't work here since the aggregator is now an interface.

Removed this change and instead created a new aggregator in this test.